inst/roxygen2-tags.yml

```diff
@@ -1,413 +1,465 @@
 - name: aliases
   description: >
-    Add additional aliases to the topic. Use `NULL` to suppress
-    the default alias automatically generated by roxygen2.
+    <p>Add additional aliases to the topic. Use <code>NULL</code> to suppress
+    the default alias automatically generated by roxygen2.</p>
   template: ' {alias}'
+  snippet: '${1:alias}'
```

Comment:
I think you can either remove `template` or replace it with the contents of snippet.

Reply:
Great, I'll do `template: ' ${1:alias}'`